### PR TITLE
ISSUE-93 - SERP Information and Preview Picture when sharing a link

### PR DESCRIPTION
**Pull request by viennt**

### CODE REVIEW

File: CHANGELOG_de-DE.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 1.5.9
```
Comment (ChristopherDosin) [inline CHANGELOG_de-DE.md:1]: Can you please move it to version `1.5.8` since this version is not released yet.

File: CHANGELOG_en-GB.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# 1.5.9
```
Comment (ChristopherDosin) [inline CHANGELOG_en-GB.md:1]: Can you please move it to version `1.5.8` since this version is not released yet.

File: composer.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
 {
     "name": "sas/blog-module",
     "description": "Blog Module",
-    "version": "1.5.8",
+    "version": "1.5.9",
```
Comment (ChristopherDosin) [inline composer.json:4]: Change back to version `1.5.8` that is not released yet.

Comment (viennt) [reply composer.json]: I changed it back. Thanks.

**Review by hungmac-sw — APPROVED** [review]

**Pull request merged**